editor/js/Sidebar.Scene.js

```diff
@@ -494,14 +494,13 @@ function SidebarScene( editor ) {
 
 	signals.refreshSidebarEnvironment.add( refreshUI );
 
-	/*
```

Comment:
I wonder why this was commented out in the first place: https://github.com/mrdoob/three.js/commit/505762636104b4e414f41b338fa72adec78096cd

Reply:
because of https://github.com/mrdoob/three.js/pull/28263#issuecomment-2092754613 ?

Reply:
fixed https://github.com/mrdoob/three.js/pull/28270